msteams-platform/m365-apps/prerequisites.md

```diff
@@ -134,6 +140,8 @@ Optionally, you can use [Visual Studio Code](https://code.visualstudio.com/) to
 
 The extension [Teams Toolkit for Visual Studio Code](https://aka.ms/teams-toolkit) (`v2.10.0` or later) provides commands that can help modify your existing Teams code to be compatible with Outlook and Microsoft 365. For more information, see [enable Teams personal tab for Microsoft 365 and Outlook](extend-m365-teams-personal-tab.md).
 
+For Word, Excel and PowerPoint Add-ins development, see [Create Office Add-in projects using Teams Toolkit](office-js-docs/docs/develop/teams-toolkit-overview.md).
```

Comment:
As this is a separate docset, relative links won't work here. Here's the format you'll want to use:

`/office/dev/add-ins/develop/teams-toolkit-overview`